mlapi/outlier.go

```diff
@@ -0,0 +1,102 @@
+package mlapi
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+)
+
+type OutlierAlgorithmConfig struct {
+	Epsilon float64 `json:"epsilon"`
+}
+
+type OutlierAlgorithm struct {
+	Name        string                  `json:"name"`
+	Sensitivity float64                 `json:"sensitivity"` // used by MAD
```

Comment:
Should this have an `omitempty` tag on it so it wouldn't be sent for Outliers?

Reply:
Comment is bad, the property is actually required for both algorithms